flink-connector-clickhouse-e2e-test/pom.xml

```diff
@@ -35,6 +35,141 @@ limitations under the License.
 	</properties>
 
 	<dependencies>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-test-utils</artifactId>
+			<version>${flink.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>clickhouse</artifactId>
+			<version>1.19.8</version>
```

Comment:
maybe can use ${testcontainer.version} replace it

Reply:
I've fixed it, please check